File: solutions/27 Remove Element [Easy]/sol.py
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        index=0
        for i in range(len(nums)):
            if nums[i]!=val:
                nums[index]=nums[i]
                index+=1
        
        return index
    
# Python    
# 29 ms (beats 94.40%)
# 16.56 mb (beats 22.24%)

File: solutions/1 Two Sum [Easy]/sol.cpp
#include <unordered_map>
 
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        unordered_map<int,int> mp;
       
        for(int i=0;i<nums.size();i++){
            if(mp.find(target-nums[i])==mp.end()) mp[nums[i]]=i;
            else return {mp[target-nums[i]],i};
        }
 
        return {-1,-1};
    }
};

// C++
// 11 ms (beats 63.38%)
// 10.82 mb (beats 99.98%)